sound/smooth_swing_v2.h

```diff
@@ -112,20 +123,34 @@ class SmoothSwingV2 : public SaberBasePassThrough {
           // that is done.
           while (A.end() < 0.0) {
             B.midpoint = A.midpoint + 180.0;
-	    Swap();
+          Swap();
           }
+          float accent_volume = 0.0;
           float mixab = 0.0;
           if (A.begin() < 0.0)
             mixab = clamp(- A.begin() / A.width, 0.0, 1.0);
-
+          
           float mixhum =
             powf(swing_strength, smooth_swing_config.SwingSharpness);
-
+          
           hum_volume =
             1.0 - mixhum * smooth_swing_config.MaximumHumDucking / 100.0;
-
+          
           mixhum *= smooth_swing_config.MaxSwingVolume;
-
+          
+          if (on_) {
+            // We need to stop setting the volume when off, or playback may never stop.
+            A.set_volume(mixhum * mixab);
+            B.set_volume(mixhum * (1.0 - mixab));
+            //This volume will scale with swing speed but is modulated by AccentSwingVolumeSharpness.
+            if (delegate_->IsSwingPlaying()) {
+              accent_volume =
```

Comment:
I kind of think we should do these volume calculations in hybrid_font.h if possible.

Reply:
ok, I can pass in the relevant variables and do it there.

Reply:
done.